announcements/2020-09-02-optional-nginx-lb-recreate-on-upgrade-to-1.8.0+ - short.md

```diff
@@ -0,0 +1,3 @@
+:information_source: There's a known issue, potential ingress downtime when upgrading optional NGINX IC App from older to v1.8.0+
+
+Please read the full notice on https://github.com/giantswarm/releases/tree/master/announcements/2020-09-02-optional-nginx-lb-recreate-on-upgrade-to-1.8.0+.md
```

Comment:
```suggestion
Please read the full notice and how to minimize impact in https://github.com/giantswarm/releases/tree/master/announcements/2020-09-02-optional-nginx-lb-recreate-on-upgrade-to-1.8.0+.md
```

Reply:
Multiple suggestions for same line - I merged one, but it invalidated this second one. I've reapplied it myself. PTAL